src/components/title-show/title-show.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom'
+import Paneset from '@folio/stripes-components/lib/Paneset';
+import Pane from '@folio/stripes-components/lib/Pane';
+import KeyValue from '@folio/stripes-components/lib/KeyValue';
+import List from '@folio/stripes-components/lib/List';
+import styles from './title-show.css';
+
+export default function TitleShow({ title }) {
+  const renderPackageListItem = item => (
+    <li key={item.packageId} data-test-eholdings-title-show-package>
+      <h5 data-test-eholdings-title-show-package-name>
+        <Link to={`/eholdings/vendors/${item.vendorId}/packages/${item.packageId}/titles/${item.titleId}`}>{item.packageName}</Link>
```

Comment:
The customer resource details made more sense to me, because you'd be maintaining your context. If it linked to the package details, then you would have to find that title all over again in the context of the package.